x/mint/keeper/hooks_test.go

```diff
@@ -50,8 +50,8 @@ func (suite *KeeperTestSuite) TestEndOfEpochMintedCoinDistribution() {
 
 		mintParams = app.MintKeeper.GetParams(ctx)
 		mintedCoin := app.MintKeeper.GetMinter(ctx).EpochProvision(mintParams)
-		expectedRewardsCoin, err := keeper.GetProportions(ctx, mintedCoin, mintParams.DistributionProportions.Staking)
-		suite.Require().NoError(err)
```

Comment:
Out of curiosity, what was the reasoning behind removing `Require().` here?

Reply:
I remember this being discussed [here](https://github.com/osmosis-labs/osmosis/pull/2046#pullrequestreview-1041071475), but i though the conclusion was to keep the `Require()`

Reply:
merge conflict - fixed :)